.github/workflows/generate.yml

```diff
@@ -0,0 +1,19 @@
+name: Generate CI
```

Comment:
```suggestion
name: CI
```

"Generate CI" might not be the most appropriate name? Also maybe rename this file to `ci.yml`?

Reply:
Done.